src/omotes_orchestrator/main.py

```diff
@@ -519,33 +525,8 @@ def main() -> None:
     if logger.isEnabledFor(logging.DEBUG):
         logger.debug("Config:\n%s", pprint.pformat(config))
 
-    workflow_type_manager = WorkflowTypeManager(
-        possible_workflows=[
-            WorkflowType(
-                workflow_type_name="grow_optimizer_default",
-                workflow_type_description_name="Grow Optimizer default workflow",
-            ),
-            WorkflowType(
-                workflow_type_name="grow_simulator", workflow_type_description_name="Grow Simulator"
-            ),
-            WorkflowType(
-                workflow_type_name="grow_optimizer_no_heat_losses",
-                workflow_type_description_name="Grow Optimizer without heat losses",
-            ),
-            WorkflowType(
-                workflow_type_name="grow_optimizer_with_pressure",
-                workflow_type_description_name="Grow Optimizer with pump pressures.",
-            ),
-            WorkflowType(
-                workflow_type_name="simulator",
-                workflow_type_description_name="High fidelity simulator",
-            ),
-            WorkflowType(
-                workflow_type_name="test_worker",
-                workflow_type_description_name="Used for testing purposes. Should not be used in "
-                "production environments.",
-            ),
-        ]
+    workflow_type_manager = WorkflowTypeManager.from_json_config_file(
```

Comment:
Not entirely clear to me why the `workflow_type_manager` is instantiated from a JSON config file here. I thought the workflow config would be centrally defined inside the `computation-engine`?

Reply:
The orchestrator reads this `../config/workflow_config.json` file which is volume mounted by the computation-engine: https://github.com/Project-OMOTES/computation-engine/blob/cb92353a12a77216945d0fccbe43e2d1ea59eba9/docker-compose.yml#L94

Reply:
Ah, okay. Does it make sense to add a short description about the purpose of `config/workflow_config.json` to README under _Directory structure_?

Reply:
Already merged this PR but it is added here: 9af8641cd23e33e2cc568b44ecde74671d3509f0